tests/jballerina-unit-test/src/test/resources/test-src/jvm/objects_subtyping.bal

```diff
@@ -161,9 +161,9 @@ public function testObjectAssignabilityBetweenNonClientAndClientObject() {
     subtyping:ClientObjectWithoutRemoteMethod o2 = new subtyping:ClientObjectWithoutRemoteMethod("ClientObjectWithoutRemoteMethod");
 
     subtyping:NonClientObject obj3 = o2;
-    subtyping:ClientObjectWithoutRemoteMethod obj4 = obj1;
+    // subtyping:ClientObjectWithoutRemoteMethod obj4 = obj1;
```

Comment:
What is the reason for commenting? I think either we have to remove the commented line or create an issue to track and uncomment it.  Otherwise, once merged, these commented code lines will remain forever.

Reply:
It was previously allowed due to a bug #43001, remove the line